app/frontend/src/features/profile/ProfileForm.tsx

```diff
@@ -55,11 +72,21 @@ export default function EditProfileForm() {
             defaultValue={user.name}
             inputRef={register}
           />
-          <ProfileTextInput
-            label="City"
+          <Controller
             name="city"
-            defaultValue={user.city}
-            inputRef={register}
+            control={control}
+            render={({ value, onChange }) => (
+              <EditUserLocationMap
```

Comment:
It's a bit annoying that when you press Enter after searching for a location, rather than submitting it to Mapbox to get the suggested list of locations, it saves what I typed as my new location instead! It'd be nice if there's an easy way to trap focus/stop the event from submitting the form when the map search input is focused.

Reply:
Yes, not sure how to get around this.

Reply:
Using the masterful art of `console.log` debugging, I've determined the `<form>`'s onSubmit is firing but never the MapSearch. So my `stopPropogation` in MapSearch isn't doing anything.

I thought the deeper nodes in the tree are meant to fire first?

Reply:
I'd think so too. Maybe it's not being focused correctly? 🤷

Reply:
Should I leave it and open a bug?

Reply:
Yeah might be worth looking at separately